examples/flask/flask_seguro/templates/base.jinja2

```diff
@@ -1,32 +1,21 @@
-<!DOCTYPE html>
-<html lang="pt-br">
-<head>
-{% block head %}
-<meta charset="utf-8">
-<meta http-equiv="X-UA-Compatible" content="IE=edge">
-<meta name="viewport" content="width=device-width, initial-scale=1">
-<title>FlaskSeguro - {% block title %}{% endblock %}</title>
-<!-- Bootstrap's CSS -->
-<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css">
-<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/app.css') }}">
-<!-- Bootstrap's Theme -->
-<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap-theme.min.css">
+{% extends 'bootstrap/base.html' %}
```

Comment:
Question: from where you're extending it? Maybe I'm lacking some flask context

Reply:
I'm extending the a flask lib, see: https://pythonhosted.org/Flask-Bootstrap/

Reply:
Oh nice, didn't know that! :smiley: